dunstctl

```diff
@@ -64,15 +72,29 @@ case "${1:-}" in
 	"history-pop")
 		method_call "${DBUS_IFAC_DUNST}.NotificationShow" >/dev/null
 		;;
-	"running")
-		property_get running | ( read -r _ _ paused; printf "%s\n" "${paused}"; )
+	"is-paused")
+                running=$(property_get running | ( read -r _ _ running; printf "%s\n" "${running}"; ))
```

Comment:
I'd say the cleanest solution here is to invert the boolean server side. No need to brainstorm too much since we're the only ones using this interface.

Just do the same running->paused change in `src/dbus.c` and invert the return values of the Set/Get functions.

What do you think?

Reply:
Ah I didn't know that these property getter and setter functions were only added together with dunstctl. I will change it on the server side then and also rename those properties from running to paused if you agree.

Reply:
It's cleaner that way so go ahead.